src/styles.scss

```diff
@@ -44,102 +44,121 @@ body {
     margin: 0 5px;
   }
 
-  // STYLING FOR TABLE LIST ENTRIES
-  .list-item {
+  // Creates helper classes for smaller and larger horizontal spacing
+  @mixin margin-lr {
+    @for $i from 1 through 10 {
+      .margin-lr-#{$i} {
+        margin: 0 #{$i}px;
+      }
+    }
+  }
+
+  @include margin-lr;
+
+  .hide {
+    display: none;
+    visibility: hidden;
+  }
+
+  .button-icon-text mat-icon {
+    margin: 0 3px;
+  }
+
+  // Style which places menu icon in upper right corner of table cell
+  .list-content-menu {
     display: grid;
-    grid-template-columns: auto 140px;
-    grid-gap: 2em;
+    grid-template-areas:
+      'hd mn'
+      'cn mn';
+    grid-template-columns: auto 50px;
+
+    .header {
+      grid-area: hd;
+      margin: 8px 0;
+    }
 
-    .list-item-info {
-      display: flex;
-      flex-direction: column;
-      justify-content: space-around;
+    .content {
+      grid-area: cn;
+    }
 
-      h3 {
-        text-transform: uppercase;
-        font-size: 1.25rem;
-      }
+    .menu {
+      grid-area: mn;
+    }
 
-      .list-item-content {
-        margin-bottom: 0.5em;
-      }
+  }
+
+  // the colored stars are placed on top of the uncolored ones
+  // the width is set dynamically with JS
+  .stars {
+    display: inline-block;
+    font-size: 1.5rem;
+    color: gray;
+    position: relative;
+    grid-area: stars;

```

Comment:
I included this in the `.list-item-rating .stars` style so if we use this component in another situation we don't need to use this `grid-area` value.  Please remove from here.